test/models/setting_test.rb

```diff
@@ -63,4 +63,13 @@ def test_class_methods_are_shorthand_for_values
 
     assert_equal(10, Setting.wishlist_size)
   end
+
+  def test_multilocale_violations_count_regulations_link
+    Setting.find_by(code: :violations_count_regulations_link).update(value: "{\"en\":\"https://example.com#some_anchor\", \"et\":\"https://example.et#some_anchor\"}")
+    I18n.with_locale(:et) { assert_equal('https://example.et#some_anchor', Setting.violations_count_regulations_link) }
+    I18n.with_locale(:en) { assert_equal('https://example.com#some_anchor', Setting.violations_count_regulations_link) }
+    Setting.find_by(code: :violations_count_regulations_link).update(value: "https://alphabetagamma.com#some_anchor")
+    I18n.with_locale(:et) { assert_equal('https://alphabetagamma.com#some_anchor', Setting.violations_count_regulations_link) }
+    I18n.with_locale(:en) { assert_equal('https://alphabetagamma.com#some_anchor', Setting.violations_count_regulations_link) }
```

Comment:
Looks cumbersome. I guess there is no other way given the current implementation of settings?

Reply:
Well, there is no need to test localisation, so let's test just if setting returns string or value of a hash k/v pair correctly.